api/mesh/v1alpha1/externalservice.proto

```diff
@@ -57,6 +57,11 @@ message ExternalService {
     }
 
     TLS tls = 2;
+
+    // If disableHostDNSEntry is set to true then a DNS entry for the external
+    // service (in the form of host:port, where host is taken from
+    // networking.address field) won't be generated
```

Comment:
Actually I believe it's a little more. There's not listener for `networking.address` no? Because atm if you set `networking.address=foo:8000`. You can reach this external service on: `<kuma.io/service>.mesh:80`  and `foo:8000` with this set to true you can only use the first option.

Reply:
This needs rewording because that's exactly what I tried to convey. So maybe:

```
    // If disableHostDNSEntry is set to true then a DNS entry for the external
    // service taken from "networking.address" won't be generated.
    // You can still reach this external service using "external-service-name.mesh:80"
    // where "external-service-name" is taken from "kuma.io/service" tag.
```

Reply:
This makes a lot more sence!